<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QWidget" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>721</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #384b06;
font: 12pt &quot;DejaVu Serif&quot;;</string>
   </property>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>581</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #9ebe41;
border-radius: 20px;</string>
    </property>
    <widget class="QLabel" name="loginTitle">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #FFF;
font-size: 40px;
font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>310</y>
       <width>100</width>
       <height>27</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #384b06;
border-radius: 5px;
color: #face23</string>
     </property>
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="usernameInput">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #face23;
border-radius: 20px;
padding-left: 15px;
padding-top: 5px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>111</width>
       <height>19</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLabel" name="incorrectCredsNotice">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #f00; 
font-weight: bold; 
background-color: rgb(56, 75, 6); 
padding-left: 10px; 
border-radius: 2px;
</string>
     </property>
     <property name="text">
      <string>incorrect credentials</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
